src/Renci.SshNet/IBaseClient.cs

```diff
@@ -10,7 +10,7 @@ namespace Renci.SshNet
     /// <summary>
     /// Serves as base class for client implementations, provides common client functionality.
     /// </summary>
-    public interface IBaseClient
+    public interface IBaseClient : IDisposable
```

Comment:
I know the interface already had a `Dispose` method, but IMO it doesn't make sense for an interface to be `IDisposable`: the interface has nothing to dispose.